packages/insured-bridge-relayer/src/RelayerHelpers.ts

```diff
@@ -35,6 +35,27 @@ export async function approveL1Tokens(
   }
 }
 
+// Iterate over provided list of whitelisted L1 tokens and remove any that are not whitelisted on a specific L2.
+// The Relayer will use this whitelist to determine which deposits to relay. By default, the whitelist will be set
+// to the list of all possible tokens provided in the RateModels dictionary. However, not all L1 tokens in this
+// dictionary will be whitelisted for all L2 deposit boxes.
+export async function pruneWhitelistedL1Tokens(
+  logger: winston.Logger,
+  l2Client: InsuredBridgeL2Client,
+  whitelistedRelayL1Tokens: string[]
+): Promise<string[]> {
+  await l2Client.update();
+  const filteredWhitelistedRelayL1Tokens = whitelistedRelayL1Tokens.filter((l1TokenAddress: string) => {
+    return l2Client.isWhitelistedToken(l1TokenAddress);
+  });
+  logger.info({
```

Comment:
I dont think this should be an `info` log. this means it'll log to slack on every run

Reply:
fixed